Epsilon.Host.WebApi/Controllers/DocumentController.cs

```diff
@@ -19,9 +19,9 @@ public DocumentController(IPageComponentManager pageComponentManager, ICompetenc
     }
 
     [HttpGet("page/{pageName}")]
-    public async Task<ActionResult<PageComponent>> GetPage(int courseId, string pageName)
+    public async Task<ActionResult<PageComponent>> GetPage(int courseId, string pageName, string userId, string name)
```

Comment:
Okay so here you must also make a choice, you can get the page by either giving a course id **or** give a user id. Course id is the simplest, but messes with the current structure. So I would suggest to try to make it work with user id, but keep in mind that you will need to find a way to determine where (in which course) pages must be stored.